src/rustdoc-json-types/lib.rs

```diff
@@ -30,7 +30,7 @@ pub type FxHashMap<K, V> = HashMap<K, V>; // re-export for use in src/librustdoc
 /// This integer is incremented with every breaking change to the API,
 /// and is returned along with the JSON blob as [`Crate::format_version`].
 /// Consuming code should assert that this value matches the format version(s) that it supports.
-pub const FORMAT_VERSION: u32 = 43;
+pub const FORMAT_VERSION: u32 = 44;
```

Comment:
After https://github.com/rust-lang/rust/pull/139393 lands, this should be rebased ontop of master, and then this should be 44->45 instead. See https://github.com/rust-lang/rust/issues/94591#issuecomment-2810339073 for details.

r=me after that.

Reply:
Shouldn't we do one release for both and therefore only increase by 1?

Reply:
rustdoc-json (the in-tree feature, not the crate) doesn't really do "releases".

Trying to merge them would rely on both PR's first appearing on the same nightly, and that's painful to organize and easy to get wrong. (More of a theoretical concern, but it'd be incorrect if someone was consuming rust from HEAD instead of nightly releases)

Reply:
Makes sense. Well, waiting for #139393 to be merged then. :)